### ISSUE TITLE
Undefined Error when moving a token.

### ISSUE THREAD

**Issue by Brimcon** [kandashi/Active-Auras#39]
Errors are thrown only on client browsers (Firefox/Chrome) Not on the Standalone Client. GM is on Standalone, Player 2 and 3 are on Chrome and Firefox respectively.
Only Active Auras is enabled. 5e 1.2.2
Firefox:
![image](https://user-images.githubusercontent.com/7198283/104856146-bbf30680-58c5-11eb-9c8f-c38fd7977190.png)

Chrome
![image](https://user-images.githubusercontent.com/7198283/104856152-c44b4180-58c5-11eb-8545-a2ad83549849.png)

**Comment by kandashi** [comment]
thanks for the report could you see if making this change to aura.js fixes the issue:
change line 231 from ```function MainAura(movedToken){

let map = new Map()```
to
```
   function MainAura(movedToken) {
        let gm = game.user === game.users.find((u) => u.isGM && u.active)
        if (!gm) return;
        
        let map = new Map();```

**Comment by Brimcon** [comment]
That fixed it!

**Comment by kandashi** [comment]
fix in #40
